network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_index_featured_video_post.html

```diff
@@ -0,0 +1,43 @@
+{% load wagtailcore_tags wagtailimages_tags i18n localization homepage_tags blog_tags static %}
+
+  <div class="tw-grid tw-grid-cols-12 tw-mb-7">
+    <div class="tw-pr-[36px]  tw-py-3  tw-flex tw-flex-col tw-col-span-12 large:tw-col-span-4 large:tw-py-0">
+      <div class="tw-flex tw-flex-wrap">
+        {% with topics=blog_page.topics.all topic_classes="tw-h6-heading tw-text-blue-80 tw-mr-2 tw-my-1" %}
+          {% for topic in topics %}
+            {% localized_version topic as localized_topic %}
+            {% get_root_or_page as parent_page %}
+            {# If we have a "root" context variable, we know this card is generated on an index page (or index page subroute) #}
+            <a class="{{ topic_classes }}" href="{% localizedroutablepageurl parent_page "entries_by_topic" topic.slug %}">{{ localized_topic }}</a>
+            {% if not forloop.last %}
+              <span class="{{ topic_classes }}"> / </span>
+            {% endif %}
+          {% endfor %}
+        {% endwith %}
+        <img src="{% static "_images/video-icon.svg" %}" alt="{% trans "video" context "icon description for screen readers" %}" class="tw-h-4 tw-w-4 tw-mr-2 tw-my-1">
+        <span class="tw-h6-heading tw-text-gray-40 tw-my-1"> {{ blog_page.first_published_at|date:"DATE_FORMAT" }} </span>
```

Comment:
**Question** Can we achieve this spacing with padding rather than actual whitespace?

Reply:
Hi @tbrlpld! I am not sure if I 100% understand your question, however, I updated the "my-1" classes to "py-1" classes to use padding instead of margin for these items. The only one I was not able to update was the image, as if I updated it to use padding, the SVG would shrink.

Reply:
Sorry if I was not clear enough here. I mean the whitespace in the span tag. Specifically between the span tag and the curly braces. 

My suggestion would have been to remove the white space and instead rely on padding (or margin) on the span element to recreate the spacing (if necessary).
```
        <span class="tw-h6-heading tw-text-gray-40 tw-my-1">{{ blog_page.first_published_at|date:"DATE_FORMAT" }}</span>
```